x-pack/plugins/security_solution/scripts/endpoint/endpoint_agent_runner/fleet_server.ts

```diff
@@ -50,21 +50,22 @@ export const runFleetServerIfNeeded = async (): Promise<
   const {
     log,
     kibana: { isLocalhost: isKibanaOnLocalhost },
-    kbnClient,
+    // kbnClient,
   } = getRuntimeServices();
 
   log.info(`Setting up fleet server (if necessary)`);
   log.indent(4);
 
-  const currentFleetServerUrl = await fetchFleetServerUrl(kbnClient);
+  // const currentFleetServerUrl = await fetchFleetServerUrl(kbnClient);
+  // const fleetServerPackagePolicy = await getFleetServerPackagePolicy();
 
-  if (currentFleetServerUrl) {
-    log.info(
-      `Fleet server is already enrolled with Fleet - URL:\n${currentFleetServerUrl}\nNothing to do.`
-    );
-    log.indent(-4);
-    return;
-  }
+  // if (currentFleetServerUrl && fleetServerPackagePolicy) {
```

Comment:
checking if `currentFleetServerUrl` exists may not be sufficient as some of the configs setup Fleet Server host on the config level, so we may end up in a state where we do have a fleet server host setup, but no actual fleer server running
https://github.com/elastic/kibana/blob/main/x-pack/test/defend_workflows_cypress/endpoint_config.ts#L37